debian_package_manager/README.md

```diff
@@ -0,0 +1,84 @@
+# debian-package-manager
+
+A debian package index parser with bazel config generation
+
+# Usage
+
+```sh
+# if new versions are available, update snapshots and bazel config
+$ go build -o update cmd/update
```

Comment:
should we just use `make update` here and have the make command run both of these?

Reply:
oh yeah good call